backend/sample.js

```diff
@@ -1,120 +1,286 @@
 const express = require("express");
 const Folder = require("./Database/Models/folderSchema");
-const {getAllFolders,
-    findFolder,
-    findNote,
-    addFolder,
-    addNote,
-    deleteFolder,
-    deleteNote} = require('./connections');
+const Note = require("./Database/Models/noteSchema");
 const app = express();
-const cors = require("cors");
+const cors = require('cors');
 const port = 5000;
+const { json } = require("express");
+const e = require("express");
+
+const folders = {
+  folderList: [
+    {
+      name: "sampleFolder",
+      notes: [{}],
+      isPrivate: false,
+      password: "",
+    },
+    {
+      name: "folder2",
+      notes: null,
+    }
+  ],
+};
 
 app.use(cors());
 app.use(express.json());
 
+async function getAllFolders() {
+  let result = await Folder.find({})
+  return result
+}
+
+async function findFolder(name) {
+  return await Folder.find({'name':name});
+}
+
+async function findNote(folderName, noteName) {
+  return await Note.find({'name':noteName, 'folder':folderName});
+}
+async function findNotesByFolder(fName){
+  return await Note.find({'folder':fName});
+}
+async function addFolder(folder) {
+  Folder.insertOne(folder);
+}
+async function deleteFolder(folderToDelete) {
+  const folder = Folder.find({'name':folderToDelete});
+  if(folder===undefined||folder.length==0){
+    return false;
+  }
+  else{
+    const folderId = folder["id"];
+    try{
+      if(Folder.findByIdAndDelete(folderId)){
+        return true;
+      }
+    }
+    catch(error){
+      console.log(error);
+      return false;
+    }
+  }
+}
+async function deleteNote(fName, noteName) {
+  const noteToDelete = Note.find({'name': noteName, 'folder':fName});
+  if(noteToDelete===undefined||noteToDelete.length==0){
+    return false;
+  }
+  else{
+    const noteId = noteToDelete["id"];
+    try{
+      if(Note.findByIdAndDelete(noteId)){
+        return true;
+      }
+    }catch(error){
+      console.log(error);
+      return false;
+    }
+  }
+}
```

Comment:
all of these async functions were just moved to connections.js. Did you git pull --rebase?